### DISCUSSION TITLE
Compression in cassandra

### DISCUSSION THREAD

**Discussion by gregsting** [glowroot/glowroot#825] [General]
Hello, 

My Cassandra DB is becoming big (100GB+) Are table compressed?

Has anybody experimented? Is there any/gain/loss with compression in performance? CPU usage? Disk usage?

Thanks for any feedback

**Comment by yasinsula061** [comment]
Selamün Aleyküm Brother,

- You need update the "auto_snapshot: true" value in the /path/to/cassandra/conf/cassandra.yaml and restart cassandra. 

if so, you can use the following commands to clean your disk.

- $_JAVA_HOME/bin/java -jar $_GLOWROOT_PATH/glowroot-central.jar truncate-all-data
- sh $_CASSANDRA_PATH/bin/nodetool listsnapshots
- sh $_CASSANDRA_PATH/bin/nodetool clearsnapshot --all
